google-cloud-storage/src/test/java/com/google/cloud/storage/BlobTest.java

```diff
@@ -341,12 +345,12 @@ public void testCopyTo() throws Exception {
 
   @Test
   public void testCopyToBlobId() throws Exception {
-    initializeExpectedBlob(2);
+    initializeExpectedBlob();
     BlobInfo target = BlobInfo.newBuilder(BlobId.of("bt", "nt")).build();
     BlobId targetId = BlobId.of("bt", "nt");
     CopyWriter copyWriter = createMock(CopyWriter.class);
     Capture<CopyRequest> capturedCopyRequest = Capture.newInstance();
-    expect(storage.getOptions()).andReturn(mockOptions);
+    expect(storage.getOptions()).andReturn(mockOptions).anyTimes();
```

Comment:
Ben in description of PR: 

```
Relax several easymock assertions on StorageOptions method. (Calling get
on an immutable object shouldn't be so concerned with invocation count)
```

Reply:
With the introduction of the RetryAlgorithmManager to StorageOptions, the number of times there are interactions with storage to get StorageOptions, or calls against an instance of StorageOptions increased. As far as test maintenence is concerned N calls to a method on an immutable object and produces an immutable result aren't interesting from a number of invocations stand point. So, rather than fighting with the numbers being unexpected, I opted to remove the constraint on the number of times the method is allowed to be called.

Reply:
Makes perfect sense, thanks for clarifying the change.